From c11bdd7e2af5b867e33b172b7b94cecb442bc554 Mon Sep 17 00:00:00 2001
From: "Guoniu.Zhou" <guoniu.zhou@nxp.com>
Date: Fri, 17 Feb 2017 12:54:11 +0800
Subject: [PATCH 1527/5242] MLK-13993: video: Distinguish fb0 and fb1 in
 userspace.

commit  c2d7fe20822bd1f5d39eda8381970172fd4ecde8 from
https://source.codeaurora.org/external/imx/linux-imx.git

In userspace, user can not distinguish fb0 and fb1, so add fb1/name property to
distinguish them.

Signed-off-by: Guoniu.Zhou <guoniu.zhou@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/video/fbdev/mxsfb.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/video/fbdev/mxsfb.c b/drivers/video/fbdev/mxsfb.c
index 989a952..5ec2341 100644
--- a/drivers/video/fbdev/mxsfb.c
+++ b/drivers/video/fbdev/mxsfb.c
@@ -1935,6 +1935,8 @@ static void init_mxsfb_overlay(struct mxsfb_info *fbi,
 	ofb->blank_state = -1;
 	ofb->global_alpha = 255;
 	ofb->fbi = fbi;
+
+	sprintf(ofb->ol_fb->fix.id, "FG");
 }
 
 static int mxsfb_overlay_map_video_memory(struct mxsfb_info *fbi,
-- 
1.7.9.5

